src/site/xdoc/cmdline.xml.vm

```diff
@@ -450,6 +450,1327 @@ java -D&lt;property&gt;=&lt;value&gt;  \
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-xml">
+&lt;?xml version="1.0"?&gt;
+&lt;!DOCTYPE module PUBLIC
+    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+    "https://checkstyle.org/dtds/configuration_1_3.dtd"&gt;
+&lt;module name="Checker"&gt;
+    &lt;module name="TreeWalker"&gt;
+        &lt;module name="FallThrough"/&gt;
+    &lt;/module&gt;
+&lt;/module&gt;
+  </code></pre>
+</div>
+
+<p><strong>Command:</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-bash">
+$ java -jar checkstyle-10.21.1-all.jar -c config.xml --exclude-regexp ".*Test.*\.java" src
+  </code></pre>
+</div>
+
+<p><strong>Test File (Test.java):</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-java">
+/**
+ * This is a sample Test class demonstrating Checkstyle functionality.
+ */
+class Test {
+    /**
+     * Demonstrates a fall-through violation in a switch statement.
+     */
+    public void foo() {
+        int i = 0;
+        while (i >= 0) {
+            switch (i) {
+                case 1:
+                case 2:
+                    i++;
+                case 3: // Violation: Fall through from previous branch.
+                    i++;
+            }
+        }
+    }
+}
+  </code></pre>
+</div>
+
+<p><strong>Output:</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-bash">
+Starting audit...
+Skipping excluded file: src/main/java/Test.java
+Audit done.
+  </code></pre>
+</div>
+
+<h4>
+15. Displaying Checkstyle Version (<span class="no-transform">-V</span>, --version)
+</h4>
+<p><strong>Goal:</strong> Print the Checkstyle version information.
+</p>
+
+<p><strong>Command:</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-bash">
+$ java -jar checkstyle-10.21.1-all.jar --version
+  </code></pre>
+</div>
+
+<p><strong>Output (Content of Version Info):</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-bash">
+Checkstyle 10.21.1
+  </code></pre>
+</div>
+
+<h4>
+16. Matching AST Branches (<span class="no-transform">-b</span>, --branch-matching-xpath)
+</h4>
+<p><strong>Goal:</strong>
+Show AST branches matching a given XPath query for the variable declaration of "x".
+</p>
+
+<p><strong>Test File (Test.java):</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-java">
+/**
+ * This is a sample Test class demonstrating Checkstyle AST matching.
+ */
+class Test {
+    /**
+     * Demonstrates extraction of AST nodes for variable definitions.
+     */
+    void foo() {
+        int x = 10;
+    }
+}
+  </code></pre>
+</div>
+
+<p><strong>Command:</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-bash">
+$ java -jar checkstyle-10.21.1-all.jar --branch-matching-xpath "/COMPILATION_UNIT
+/CLASS_DEF/OBJBLOCK/METHOD_DEF/SLIST/VARIABLE_DEF[./IDENT[@text='x']]" Test.java
```

Comment:
this command is not copy-paste able.

can we use just code block to let it be rendered differently,
see example table of CLI options.
```
<code>-E, --executeIgnoredModules</code>
```